spec/support/page_objects/support/user_show_page.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Support
+    class UserShow < PageObjects::Base
+      set_url "/support/users/{id}"
+
+      sections :provider_rows, PageObjects::Sections::Provider, ".qa-provider_row"
```

Comment:
This had me stumped for a while - I originally called these sections `:providers` but that conflicted with api request specs that defined `@providers`. For example:

```
        before do
          @providers = Array.new(25) do |i|
            create(:provider, provider_code: "PROV#{i + 1}",
                   changed_at: (30 - i).minutes.ago,
                   sites: [])
          end
        end
```

in `spec/requests/api/v1/provider_spec.rb` 😱 

I've just changed the name of to `provider_rows` to avoid the conflict, but I don't feel great about the underlying!